crates/rnote-ui/data/ui/mainheader.ui

```diff
@@ -40,6 +40,11 @@
                     <property name="tooltip_text" translatable="yes">Show/Hide Sidebar</property>
                   </object>
                 </child>
+                <child>
+                  <object class="AdwTabButton" id="overview_button">
+                    <property name="action-name">overview.open</property>
+                  </object>
+                </child>
```

Comment:
Is there any better place for the overview button?

IMHO it might seem confusing to have the new-tab button right beside the overview button.

As an idea, we can add it as a `start-action-widget` or `end-action-widget` in the `TabBar`.

Reply:
what about changing the icon to the one other apps like ptyxis use? I think it would make it more clear that this will open an tab overview.

![Neues Dokument - Selection](https://github.com/user-attachments/assets/b825b80b-5232-49b8-a5cb-7b4fdcc626d6)

Reply:
It looks great, and anyway there is a tab counter inside the overview, so we can still see how many tabs are opened.